doc/specs/vulkan/chapters/memory.txt

```diff
@@ -1721,6 +1721,7 @@ ifdef::VK_KHX_device_group[]
   * [[VUID-vkMapMemory-memory-00683]]
     pname:memory must: not have been allocated with multiple instances.
 endif::VK_KHX_device_group[]
+  * pname:ppData must: be a valid pointer to a pointer
```

Comment:
PS Actually e.g. `vkUpdateDescriptorSetWithTemplateKHR` is a freak case too. Generates just:

> `pData` **must** be a pointer

That just sounds wrong.

Perhaps the `vk.xml` parsing script should throw error/exception when there is a `void*` without `len` && without `noautovalidity.